vuepress/docs/next/tutorials/devops/backing-up-and-restoring-your-environment.md

```diff
@@ -4,118 +4,96 @@ redirectFrom: /next/tutorials/customize-the-platform/app-engine/building-prepack
 
 # Backing Up and Restoring an Entando Application
 
-## Prerequisites
-
--   Java 11
-
--   Docker installed locally
-    (<https://docs.docker.com/docker-for-windows/install/>)
+## Overview
 
--   maven
+This tutorial demonstrates how to use an existing Entando application as a launching point for creating new applications. An alternative is to [use the Entando Bundler](../ecr/export-bundle-from-application.md) to extract specific content and functionality from an existing application. Entando Bundles are more flexible and portable so their use is recommended when a full copy of an application is not required.
```

Comment:
...should the preceding "Entando" make "application" a proper noun + capitalized, like "Entando Bundles"?

pretty sure add comma: "...portable, so..."